SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Android/Analytics.cs

```diff
@@ -47,6 +47,22 @@ public static Task SetEnabledAsync(bool enabled)
             return Task.Run(() => future.Get());
         }
 
+        /// <summary>
+        /// Pause the Analytics service.
+        /// </summary>
+        public static void Pause()
```

Comment:
This is a new API and thus a new feature.
The problem is that we don't have a Windows implementation for now and we would need to update 1DS and appcenter docs for it.

Let's make it internal for now until it's officially a feature. (internals are visible to test assembly)
```suggestion
        internal static void Pause()
```